IPTables.Net.Tests/IpTableSystemTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using IPTables.Net.Iptables;
+using IPTables.Net.Iptables.Adapter;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SystemInteract.Local;
+
+namespace IPTables.Net.Tests
+{
+    [TestFixture]
+    internal class IpTableSystemTests
+    {
+        [SetUp]
+        public void Initialize()
+        {
+            if (IsLinux)
+            {
+                if (Environment.GetEnvironmentVariable("SKIP_SYSTEM_TESTS") != "1")
+                {
+                    _system = new IpTablesSystem(system: new LocalFactory(), tableAdapter: new IPTablesBinaryAdapter());
+                }
+            }
+        }
+
+        [Test]
+        public void TestGetRules()
+        {
+            if (IsLinux)
+            {
+                if (Environment.GetEnvironmentVariable("SKIP_SYSTEM_TESTS") == "1")
+                {
+                    Assert.Ignore();
+                }
+
+                // Invalid table cause exception
+                Assert.Throws<ArgumentException>(() => _system.GetRules("INPUT", IP_VERSION));
```

Comment:
Refrain from using a common chain name as a table name, use `_invalidTableName` or something

Reply:
Ok

Reply:
I do understand your mistake though :)

Reply:
Also probably throwing an iptables exception would be better to keep with style.

Reply:
Did you want add `IpTablesNetArgumentException` class?

Reply:
@splitice

Reply:
I think `ArgumentException` is enough and good fit for this situation, but if you want, I can create a new class that name is `IpTablesNetArgumentExpcetion` and throw this class instead of standard A`rgumentException`, please make decision and inform me, to apply changes as your decision.

Reply:
@splitice